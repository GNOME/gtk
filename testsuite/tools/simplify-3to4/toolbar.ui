<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk+" version="3.0"/>
  <object class="GtkToolbar">
    <property name="visible">1</property>
    <property name="toolbar_style">both</property>
    <child>
      <object class="GtkToolButton">
        <property name="visible">1</property>
        <property name="label" translatable="yes">New</property>
        <property name="icon-name">document-new</property>
        <property name="tooltip-text" translatable="yes">Create a new document</property>
      </object>
      <packing>
        <property name="homogeneous">1</property>
      </packing>
   </child>
   <child>
     <object class="GtkSeparatorToolItem">
        <property name="visible">1</property>
     </object>
   </child>
   <child>
     <object class="GtkToggleToolButton">
     </object>
   </child>
  </object>
</interface>
