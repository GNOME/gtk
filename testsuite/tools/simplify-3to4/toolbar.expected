<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkBox">
    <property name="css-classes">toolbar</property>
    <child>
      <object class="GtkButton">
        <property name="label" translatable="1">New</property>
        <property name="icon-name">document-new</property>
        <property name="tooltip-text" translatable="1">Create a new document</property>
      </object>
    </child>
    <child>
      <object class="GtkSeparator">
        <property name="orientation">vertical</property>
      </object>
    </child>
    <child>
      <object class="GtkToggleButton"/>
    </child>
  </object>
</interface>
